benchmark/infrastructure/BenchmarkMeasurementContainer.h

```diff
@@ -249,6 +249,13 @@ class ResultTable : public BenchmarkMetadataGetter {
   */
   void addRow();
 
+  /*
+  @brief Delete the given row.
+
+  @param row Uses matrix coordinates.
+  */
+  void deleteRow(const size_t row);
```

Comment:
better call the argument `rowIdx` that makes it even clearer.

Reply:
Done, see [0247add](https://github.com/ad-freiburg/qlever/pull/1207/commits/0247add25a431fc066c2037eb2b1574bd8e859f0).